modules/transformer-kinesis/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/kinesis/Main.scala

```diff
@@ -32,10 +35,24 @@ object Main extends IOApp {
       parsed <- ShredderCliConfig.Stream.loadConfigFrom[IO](BuildInfo.name, BuildInfo.description)(args: Seq[String]).value
       res <- parsed match {
         case Right(cliConfig) =>
-          Resources.mk[IO](cliConfig.igluConfig, cliConfig.config.queue, cliConfig.config.monitoring.metrics).use { resources =>
+          Resources.mk[IO](
+            cliConfig.igluConfig,
+            cliConfig.config.queue,
+            cliConfig.config.monitoring.metrics,
+            executionContext
+          ).use { resources =>
+            val telemetry =
+              Telemetry.run[IO](
+                cliConfig.config,
+                Some(Telemetry.Cloud.Aws),
```

Comment:
We're in `transformer.kinesis` package so I assumed that the cloud would always be AWS, but that's right that it's possible to run the transformer locally, although it would still send messages either to SQS or SNS, so in AWS.